### ISSUE TITLE
[DX] Save Dev time, make a console notice for Backdrop.behaviours

### ISSUE THREAD

**Issue by YesCT** [backdrop/backdrop-issues#4116]
Lots of folks have their computer language set to British English, and their editors autocomplete Backdrop.behaviors to Backdrop.behaviours this can cause a lot of frustration and time lost.

Log a notice to the console like:
Sorry for our "behavior". -American ubiquity

Alternatively,
Less "funny":
Backdrop uses American spelling, for example in Backdrop.behaviors

Credit: justafish

**Comment by klonos** [comment]
Yeah, I always find it funny how people "lecture" me here in OZ about "proper" English; but they are Drupal devs, so they have to use the color.module - not the colour.module 😝

Jokes aside, how are our Drupal brethren dealing with this issue?
